daemon/buildinfo.c

```diff
@@ -312,3 +313,49 @@ void print_build_info_json(void) {
     printf("  }\n");
     printf("}\n");
 };
+
+//return a list of enabled features for use in analytics
+//find a way to have proper |
+void analytics_build_info(BUFFER *b) {
+    if(FEAT_DBENGINE)        buffer_strcat (b, "dbengine");
```

Comment:
Considering what I saw in this function, we are having pipe as separator, and we are assuming that `dbengine` will be always enabled, and thanks this, this is the unique keyword without a pipe. What will happen on webserver side when `dbengine` is not enabled and the message starts with a pipe `|JSON-C`, for example?

Reply:
Hi Thiago. Yes, I know it's not optimal, but I believe posthog will handle it (I was going to  talk to @andrewm4894 about this). Basically from what I know, posthog can build it's own internal list separating by the pipe. An initial empty entry (i.e. without `dbengine`) could be handled by posthog. If not, I will add a counter there.

Reply:
I wrote a consideration for integer size, but it is something small, this is the greatest concern I have right now, I will wait for @andrewm4894 confirmation before to do the last test and to approve your PR.

Reply:
yep something like "|foo|bar||blah|" would be ok by me - the posthog plugin will then downstream use javascript to split the string by "|" and just ignore the nulls in the resulting array. 

there is an example [here](https://github.com/netdata/posthog-netdata-event-processing/blob/main/process_properties.js#L25) that shows what i mean in the data processing plugin code.